cmd/flags/common.go

```diff
@@ -128,6 +128,12 @@ var (
 		Category: commonCategory,
 		Value:    12 * time.Second,
 	}
+	// ConfigFile Path to a config file
+	ConfigFile = &cli.StringFlag{
+		Name:     "config.file",
```

Comment:
Let's use `config` here (and change the flag description), to keep the same as what go-ethereum is using:

<img width="520" alt="image" src="https://github.com/taikoxyz/taiko-client/assets/104078303/5c28c93c-955f-4a37-84b3-21fbe5c3bc96">

Reply:
ok